scripts/version-utils.js

```diff
@@ -0,0 +1,144 @@
+  const filePath = path.resolve(__dirname, '../CHANGELOG.md');
+  let changeLog = fs.readFileSync(filePath).toString();
+  const latestEntryIndex = changeLog.indexOf('## [');
+  const newEntry = `## [${newVersion}] - ${new Date().toISOString().slice(0, 10)}
+    \n### Added
+    \n### Removed
+    \n### Changed
+    \n### Fixed
+    \n`;
+  changeLog = changeLog.substring(0, latestEntryIndex) + newEntry + changeLog.substring(latestEntryIndex);
+  fs.writeFileSync(filePath, changeLog);
+};
+
+// Update the base branch to point to a new branch.
+// For example, base branch for release-2.x should be origin/release-2.x
+const updateBaseBranch = (branchName) => {
+  logger.log(`Updating the base branch in .base-branch to ${branchName}`);
+  const filePath = path.resolve(__dirname, '../.base-branch');
+  fs.writeFileSync(filePath, `origin/${branchName}`);
+}
+
+const versionBump = (option, branchName) => {
+  if (!option) {
+    logger.log('Choose one of the following options to bump the next version:');
+    logger.log('  1. Patch');
+    logger.log('  2. Minor');
+    logger.log('  3. Major');
+    logger.log('  4. Beta');
+    option = Number(prompt(''));
+  }
+  const newVersion = getNewVersion(currentVersion, option);
+  if (!newVersion) {
+    process.exit(1);
+  }
+
+  branchName = branchName ? branchName : 'version-bump';
+
+  const prevReleaseBranch = !isPreRelease(currentVersion) && (option === 3 || option === 4)
+    ? `release-${currentVersion.split('.')[0]}.x`
+    : '';
+
+  logger.warn('Warning: you are bumping the version\n');
+  logger.warn(`  From: ${currentVersion}\n`);
+  logger.warn(`  To: ${newVersion}\n`);
+  if (prevReleaseBranch) {
+    logger.warn(`  This will also create ${prevReleaseBranch} branch.`);
+  }
+  shouldContinuePrompt();
+
+  if (prevReleaseBranch) {
+    const currentBranch = (spawnOrFail('git', [' branch --show-current'], { skipOutput: true })).trim();
+    spawnOrFail('git', [`checkout -b ${prevReleaseBranch}`]);
+    updateBaseBranch(prevReleaseBranch);
+    spawnOrFail('git', ['add -A']);
+    spawnOrFail('git', [`commit -m "Update base branch to ${prevReleaseBranch}"`]);
+    spawnOrFail('git', [`push origin HEAD:${prevReleaseBranch} -f`]);
+    logger.log(`Branch ${prevReleaseBranch} is created. Please make sure to set branch protection.`);
+
+    // Switch back to the local branch
+    spawnOrFail('git', [`checkout ${currentBranch}`]);
+  }
+
+  spawnOrFail('npm', [`version ${newVersion} --no-git-tag-version`]);
+  updateChangelog(newVersion);
+
+  logger.log('Committing version bump...');
+  spawnOrFail('git', ['add -A']);
+  spawnOrFail('git', [`commit -m "Version bump for amazon-chime-sdk-js@${newVersion}"`]);
+  logger.log(`Do you want to upload these files to ${branchName} branch?\n`);
+  shouldContinuePrompt();
+  spawnOrFail('git', [`push origin HEAD:${branchName} -f`]);
+  logger.log('Please create a pull request to merge the version bump to main.');
```

Comment:
is there something besides logger.log that will make this show in bold? Otherwise its easy to miss.

Reply:
We can change to log.warn() but this will be a step in the MCM so not sure if it is needed.